.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: docker release
```

Comment:
(nitpick)

I also think that the `name` (as well as the file name) is slightly misleading because CI jobs in this workflow do not publish any Docker releases. Instead this CI job is related to applying correct tags to the already released versions.
I guess we could change this `name` to something like "**Apply docker tags to the official JanusGraph release**".
I would also probably rename this file from `docker-release.yml` to something like `docker-release-tags.yml`.